app/src/main/kotlin/com/wire/android/ui/userprofile/self/SelfUserProfileScreen.kt

```diff
@@ -335,15 +337,28 @@ private fun OtherAccountsHeader() {
 }
 
 @Composable
-private fun NewTeamButton(onAddAccountClick: () -> Unit) {
+private fun NewTeamButton(onAddAccountClick: () -> Unit, isUserIdCall: () -> Boolean) {
+    val context = LocalContext.current
```

Comment:
I remember @trOnk12 always trying to bring this composable to the top parent, and pass it down the stream.
Maybe we could do something similar and have it as the rest of our composables ? :D

Reply:
Done !